requirements-app.txt

```diff
@@ -43,7 +43,7 @@ git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd
 # The production image defined by Dockerfile uses Gunicorn.  The development image defined by Dockerfile.local
 # runs the application with the Flask development server by default.
 # 9 May 2022: Newer versions cause build or runtime issues.
-gunicorn[eventlet] < 20.1.0
+gunicorn[eventlet] @ git+https://github.com/benoitc/gunicorn.git@6a8ebb4844b2f28596ffe7421eb9f7d08c8dc4d8
```

Comment:
Please document why this needs to be pinned to a specific commit.

Reply:
Done.